docs/networking/clusternetwork.md

```diff
@@ -80,3 +104,9 @@ You are allowed to add the custom cluster network, which will not be available u
 - The text `enp7s3 (1/3 Down)` in the NICs drop-down list indicates that the enp7s3 NIC is down in one of the three selected nodes. In this case, you need to find the NIC, set it up, and refresh this page. After this, it should be selectable.
 
 :::
+
+:::note
+
+Starting with Harvester v1.1.2, Harvester supports updating network configs. Make sure to stop all affected VMs before updating network configs.
```

Comment:
@yaocw2020, we may need to mention the upgrade issue of https://github.com/harvester/harvester/issues/3223 here and say that it will get addressed in v1.1.2. WDUT? Thanks.

Reply:
Say it will get addressed in v1.1.2 in the issue or document?